h/api/presenters.py

```diff
@@ -54,7 +54,8 @@ def tags(self):
 
     @property
     def target(self):
-        target = {'source': self.annotation.target_uri}
+        target = {'source': self.annotation.target_uri,
+                  'scope': self.annotation.target_uri_normalized}
```

Comment:
To be clear, I hope that eventually we don't use the presenters at all for search indexing -- the requirements are fundamentally different. It's a nice shortcut at the moment because it preserves (mostly) the format our search index is already in, but I'd like us to avoid having to unpick two different sets of requirements from the presenters in the future. So if there's stuff that only needs to be in the search index, it shouldn't be in the presenters.